core/domain/exp_domain.py

```diff
@@ -1336,7 +1336,7 @@ def can_undergo_classification(self):
         return False
 
     def update_content(self, content_dict):
-        """Update the list of Content of this state.
+        """Update the list of content of this state.
```

Comment:
Ditto.

Reply:
@vojtechjelinek, have you gone through this? "Update the `list` of content of this state."

Reply:
Done.